designer/deployment-manager-api/src/main/scala/pl/touk/nussknacker/engine/DeploymentManagerDependencies.scala

```diff
@@ -2,20 +2,17 @@ package pl.touk.nussknacker.engine
 
 import cats.effect.unsafe.IORuntime
 import org.apache.pekko.actor.ActorSystem
-import pl.touk.nussknacker.engine.api.component.{ComponentAdditionalConfig, DesignerWideComponentId}
-import pl.touk.nussknacker.engine.api.deployment.{ProcessingTypeActionService, ProcessingTypeDeployedScenariosProvider}
+import pl.touk.nussknacker.engine.api.deployment.ProcessingTypeDeployedScenariosProvider
 import sttp.client3.SttpBackend
 
 import scala.concurrent.{ExecutionContext, Future}
 
 case class DeploymentManagerDependencies(
     deployedScenariosProvider: ProcessingTypeDeployedScenariosProvider,
-    actionService: ProcessingTypeActionService,
```

Comment:
Removed dependencies were only for scheduling mechanism purpose - I moved them there